subsys/bluetooth/mesh/net.c

```diff
@@ -57,7 +57,7 @@
 /** Define CONFIG_BT_MESH_SEQ_STORE_RATE even if settings are disabled to
  * compile the code.
  */
-#ifndef CONFIG_BT_SETTINGS
+#if !defined(CONFIG_BT_SETTINGS) || !defined(CONFIG_BT_MESH_CONTINUOUS_STORE)
```

Comment:
Doesn't `!defined(CONFIG_BT_MESH_CONTINUOUS_STORE)` cover both, since the option depends on `BT_SETTINGS`?